src/ontodia/diagram/view.ts

```diff
@@ -432,9 +432,122 @@ export class DiagramView extends Backbone.Model {
                 end: this.createLinkMarker(linkTypeId, false, style.markerTarget),
             };
         }
+        style.router = this.getLinkRouter();
         return style;
     }
 
+    private getLinkRouter(): LinkRouter {
+        return (vertices: Vertex[], args: RouterProps, linkView: LinkView): Vertex[] => {
+            const link = linkView.model;
+            if (!vertices) {
+                vertices = [];
+            }
+            // If the cell is a view, find its model.
+
+            // The cell is a link. Let's find its source and target models.
+            let srcId = link.get('source').id || link.previous('source').id;
+            let trgId = link.get('target').id || link.previous('target').id;
+
+            // Use the same direction for all siblings
+            const direction = srcId > trgId;
+            if (direction) {
+                const temp = srcId;
+                srcId = trgId;
+                trgId = temp;
+            }
+
+            // If one of the ends is not a model, the link has no siblings.
+            if (!srcId || !trgId) {
+                return vertices;
+            }
+
+            const links: Link[] = getLinksBetweenElements (this.model, srcId, trgId);
+            if (links.length < 2) {
+                return vertices;
+            }
+
+            const siblings: Link[] = links.filter((l) => {
```

Comment:
Please don't use `l` as parameter name (it's easy to confuse it with `1`, `I` or `|`).